docs/dev/win_dev.rst

```diff
@@ -9,15 +9,4 @@ creates a copy of the data files instead of using symbolic links.
 As a result, if you are changing the contents in ``share`` folder
 under the source checkout's root directory, you will need to rebuild
 the package even in editable install mode. Otherwise, this should not
-affect your development experience.
-
-WSL2 and voila
```

Comment:
Unless Solara is super magical, I bet this is still a problem.

Reply:
can anyone test this?  It would be nice to get rid of this if it isn't applicable anymore, but if it is, I'm happy to re-introduce it and just swap out voila>solara.

Reply:
I can test it on Monday if no one else has.

Reply:
I just checked. Still more or less the same problem. I still have to manually copy paste the server URL to browser when I run server from WSL2 but the browser is on Windows.

```
Solara server is starting at http://localhost:8765
gio: http://localhost:8765: Operation not supported
```

Reply:
ok, I'll revert and just change voila>solara then.  Thanks for checking